src/prisma/schema.prisma

```diff
@@ -254,6 +254,7 @@ model Space {
   name                        String
   domain                      String                            @unique
   customDomain                String?                           @unique
+  isCustomDomainVerified      Boolean?                          @default(false)
```

Comment:
this is the change, rest is result of `prisma format` command

Reply:
I would have just let 'null' be the default, and it only turns false if, say, it was verified and then it wasn't. but it doesn't really matter